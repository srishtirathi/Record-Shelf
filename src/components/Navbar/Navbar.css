nav {
    background-color: #38497E;
    display: flex;
    margin: 0;
    justify-content: space-between;
    padding: 1 rem;
    color: white;
}