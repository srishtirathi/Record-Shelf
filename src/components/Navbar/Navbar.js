import React from 'react';
import './Navbar.css';

const Navbar = () => (
  <div>
    <nav>
      <h1>
        My Record Shelf
      </h1>
    </nav>
  </div>
);
export default Navbar;
