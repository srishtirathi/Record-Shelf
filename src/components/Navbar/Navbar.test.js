/* eslint-disable react/jsx-props-no-spreading */
import { render, screen } from '@testing-library/react';
import React from 'react';
import {
  BrowserRouter,
} from 'react-router-dom';
import Navbar from './Navbar';

describe(Navbar.name, () => {
  afterEach(() => {
    jest.clearAllMocks();
  });
  it('sanity check', () => {
    expect(1 + 1).toBe(2);
  });
  it('should match snapshot', () => {
    const { container } = render(<BrowserRouter><Navbar /></BrowserRouter>);
    expect(container).toMatchSnapshot();
  });
  it('should render text ', () => {
    render(<BrowserRouter><Navbar /></BrowserRouter>);
    screen.getByText('My Record Shelf');
  });
});
