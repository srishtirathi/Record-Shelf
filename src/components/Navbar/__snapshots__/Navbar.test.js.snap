// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar should match snapshot 1`] = `
<div>
  <div>
    <nav>
      <h1>
        My Record Shelf
      </h1>
    </nav>
  </div>
</div>
`;
